src/services/models/iotHubManagerModels.js

```diff
@@ -24,7 +24,7 @@ export const toDeviceModel = (device = {}) => {
     'c2DMessageCount': 'c2DMessageCount',
     'enabled': 'enabled',
     'lastStatusUpdated': 'lastStatusUpdated',
-    'iotHubHostName': 'iotHubHostName',
+    'ioTHubHostName': 'ioTHubHostName',
```

Comment:
Isn't this naming convention wrong? iotHubHostName is camel case no idea what ioTHubHostName is :)

Reply:
ioTHubHostName is what our code will translate as "camel case" from IoTHubHostName, which is what is returned. We can map ioTHubHostName to iotHubHostName in this and use that as the variable name

Reply:
Yeah can we do that keep client side iotHubHostName?